ToDoApp/containers/home/Home.js

```diff
@@ -34,32 +35,77 @@ var data = [
         'second-description': 'Second Description',
         'status': false
     }
-];
-
+];*/
 class Home extends Component {
+  constructor(props){
+      super(props);
+      this.state = {
+          data:[{
+            'first-description': 'Description',
+            'second-description': 'Second Description',
+            'status': false
+        },
+        {
+            'first-description': 'Description',
+            'second-description': 'Second Description',
+            'status': false
+        },
+        {
+            'first-description': 'Description',
+            'second-description': 'Second Description',
+            'status': false
+        },
+        {
+            'first-description': 'Description',
+            'second-description': 'Second Description',
+            'status': true
+        }, 
+        {
+            'first-description': 'Description',
+            'second-description': 'Second Description',
+            'status': false
+        }]
+    }
+    StatusBar.setBackgroundColor('#1f86ff');
```

Comment:
Instead of calling this function, it's better to _render_ the `StatusBar` in JSX and feed the background color as a prop `<StatusBar backgroundColor="#1f86ff"} />`